tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "resolveJsonModule": true,
     "moduleResolution": "Node",
     "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
what is the reason for this change?

Reply:
Looks like the current "stable" version of opengsn is kind of broken, here's what happens without it https://forum.opengsn.org/t/typescript-compiler-fails-with-opengsn-v2-2-1/169